### ISSUE TITLE
Install the psi4

### ISSUE THREAD

**Issue by xiki-tempula** [openforcefield/openff-bespokefit#162]
Hi, I was wondering how should I install the psi4 with [openff-bespokefit](https://github.com/openforcefield/openff-bespokefit)?
I did this via
```
mamba create -n bespokefit -c conda-forge openff-bespokefit python=3.9
conda install -c conda-forge -c defaults -c psi4 psi4
```
I used mamba in the first part as it would otherwise be too slow.
But running psi4 will give
```
Traceback (most recent call last):
  File "/home/ec2-user/miniconda3/envs/bespokefit/lib//python3.9/site-packages/psi4/__init__.py", line 55, in <module>
    from . import core
ImportError: /home/ec2-user/miniconda3/envs/bespokefit/lib//python3.9/site-packages/psi4/core.cpython-39-x86_64-linux-gnu.so: undefined symbol: __svml_exp4_mask_e9

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ec2-user/miniconda3/envs/bespokefit/bin/psi4", line 183, in <module>
    import psi4  # isort:skip
  File "/home/ec2-user/miniconda3/envs/bespokefit/lib//python3.9/site-packages/psi4/__init__.py", line 60, in <module>
    raise ImportError("{0}".format(err))
ImportError: /home/ec2-user/miniconda3/envs/bespokefit/lib//python3.9/site-packages/psi4/core.cpython-39-x86_64-linux-gnu.so: undefined symbol: __svml_exp4_mask_e9
```
I wonder if I can have some help on this, please? Thank you.
This is a regular linux x86 machine.

**Comment by SimonBoothroyd** [comment]
@xiki-tempula unfortunately installing `psi4` can be a bit flaky due to it using a different build ecosystem than `conda-forge`, but I've found that:

```
mamba create -n bespokefit -c psi4 -c conda-forge -c defaults python=3.9 openff-bespokefit psi4
```

usually works and if I remember correctly resolves the error you are seeing.

**Comment by xiki-tempula** [comment]
@SimonBoothroyd Thanks, this does solve the psi4 issue but would break the bespokefit.
If I run `openff-bespoke executor run --file "lig.sdf" --workflow "default" --output "lig.json" --output-force-field "lig.offxml" --n-qc-compute-workers 2 --qc-compute-n-cores 8 --default-qc-spec psi4 B3LYP-D3BJ DZVP`, I got

**Comment by SimonBoothroyd** [comment]
Apologies for this @xiki-tempula, I think you've probably run into [another known issue](conda-forge/h5py-feedstock/issues/103).. the good news is you should be able to fix this by running:

```
mamba install "h5py <3.2"
```

**Comment by xiki-tempula** [comment]
Thanks.

**Comment by jthorton** [comment]
Lets get the env create command added to the docs to help future users.

**Comment by j-wags** [comment]
I think Simon's command above works for users with OpenEye, but it doesn't pull in AmberTools (and therefore fails at fragmentation). For the quickstart guide I'm going to try to find a way to make an env with AmberTools.

Jotting this down here as a confirmed pathway while I look for others- One thing that just worked on my mac is going without psi4:
```
mamba create -n bespokefit -c conda-forge -c defaults openff-bespokefit xtb-python openff-toolkit
conda activate bespokefit
```
and the following bespokefit run finished in ~20 minutes
```
openff-bespoke executor run --smiles "CC(=O)NC1=CC=C(C=C1)O" --workflow "default" --output "acetaminophen.json" --output-force-field "acetaminophen.offxml" --n-qc-compute-workers 2 --qc-compute-n-cores   8 --default-qc-spec xtb gfn2xtb none
```

**Comment by j-wags** [comment]
After some more digging (and this is probably something that others have told me multiple times) I'm moving forward with the assumption that AmberTools and Psi4 hate each others' fortran compilers in mac envs, and that this is irresolvable. 

So, I think what we should do is change the quickstart guide to recommend using XTB, and loudly say "hey! we don't think this gives as accurate results as using Psi4, which is what we used in our shown results! And if you want to use Psi4 then you must be on Linux (or if you're patient you can use a linux docker container on your mac!)"

I'll work on a PR to update the quickstart guide with that info this afternoon.

**Comment by mattwthompson** [comment]
I believe this has been resolved (#184) although the underlying continues to pop up #210
